### PR TITLE
New package: papis-0.8

### PR DESCRIPTION
**Pull request by xaltsc**
_Hello papi mais qué pasa ?_ 🎵

There are unsatisfied dependencies I'm working on to resolve right now. Also, there are outdated dependencies.

This pacakge depends on python-prompt_toolkit-2. Do not merge it until it is updated and until the other packages are merged.

### CODE REVIEW

File: srcpkgs/papis/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'papis'
+pkgname=papis
+version=0.8
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-requests python3-yaml python3-chardet python3-BeautifulSoup4 python3-colorama python3-click python3-slugify python3-prompt_toolkit python3-tqdm python3-Pygments python3-stevedore"
```
Comment (maxice8) [inline srcpkgs/papis/template]: please cut the line at 80 columns (before it if the 80th column is not space)

**Comment by maxice8** [comment]
You can take a look at setup.py to see what dependencies the package has

**Comment by xaltsc** [comment]
Yes, I've done this, I'm packaging them.

**Comment by xaltsc** [comment]
Should be ready to merge as soon as #10633 and #10620 are merged.

File: srcpkgs/papis/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'papis'
+pkgname=papis
+version=0.8
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-requests python3-yaml python3-chardet python3-BeautifulSoup4
+python3-colorama python3-click python3-slugify python3-prompt_toolkit
```
Comment (maxice8) [inline srcpkgs/papis/template]: 1 space on each line indented

Comment (xaltsc) [reply srcpkgs/papis/template]: done.

**Comment by kozachynskyi** [comment]
Installation fails with given text:

MISSING: python3-requests>2.20.9<2.21.1
Transaction aborted due to unresolved dependencies.

I have python3-requests-2.22.0_2 installed.

**Pull request merged**